docs-mslearn/fabric/create-finops-workspace.md

```diff
@@ -0,0 +1,235 @@
+
+>[!IMPORTANT]
+>You must have the *Storage Blob Data Contributor* role when you use *Organizational account* authentication. If you don't have the required permission, you can use other authentication types like *Account key*. Otherwise, you get an `Invalid credentials` error.
+
+To get the Data Lake Storage URL, view the storage account where the export created a directory and the FOCUS cost file. Under **Settings**, select **Endpoints**. Copy the URL marked as **Data Lake Storage**. 
+
+Here’s an example screenshot showing the Data Lake Storage URL on the **Endpoints** page.
+
+:::image type="content" source="./media/create-fabric-workspace-finops/endpoints-page-storage-account.png" border="true" alt-text="Screenshot showing the Endpoints page of the storage account." lightbox="./media/create-fabric-workspace-finops/endpoints-page-storage-account.png" :::
+
+## Copy data into Fabric
+
+After the shortcut gets created, you can view the FOCUS cost data inside **Files**. You can load the data directly into a Fabric table by using the following steps.
+
+1. In the lakehouse, find the directory that you created when you set up the export. The Directory is in the **Files** section.
+2. Next to the directory, select the ellipsis (**...**), and then select **Load to Tables** > **New table**.  
+    :::image type="content" source="./media/create-fabric-workspace-finops/load-to-tables-new-table.png" border="true" alt-text="Screenshot showing the New table option." lightbox="./media/create-fabric-workspace-finops/load-to-tables-new-table.png" :::
+    1. Table name = &lt;*Any valid name*&gt;
+    2. File type = `CSV`
+    3. Including subfolders = **Enabled**
+    4. Use header for column names = **Enabled**
+    5. Separator = `,`
+    6. Select **Load**
+
+:::image type="content" source="./media/create-fabric-workspace-finops/load-folder-to-new-table.png" border="true" alt-text="Screenshot showing the new table options." lightbox="./media/create-fabric-workspace-finops/load-folder-to-new-table.png" :::
+
+For more information, see [Lakehouse Load to Delta Lake tables](/fabric/data-engineering/load-to-tables).
+
+This process creates a table based on the CSV/Parquet file and you can transform data using Power Query in the report. For an automated process to ingest data using notebooks, see [Automate data ingestion](#automate-data-ingestion) later in this article.
+
+Here’s an example screenshot showing data in the Lakehouse table.
+
+:::image type="content" source="./media/create-fabric-workspace-finops/fabric-load-table-lakehouse.png" border="true" alt-text="Screenshot showing data in the table." lightbox="./media/create-fabric-workspace-finops/fabric-load-table-lakehouse.png" :::
+
+## Create a Power BI report
+
+After the data is ingested into Fabric and Delta parquet tables are ready, you can move on to reporting.
```

Comment:
No need to mention "delta parquet". They won't know (or need to know) what that means.
```suggestion
After the data is ingested into Fabric and tables are ready, you can move on to reporting.
```